libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -296,8 +296,15 @@ class AP_BoardConfig {
         float output;
         uint32_t last_log_ms;
         float temperature;
-        AP_Int8 imu_arming_temperature_margin_low;
     } heater;
+
+    void heater_update_and_log(struct imu_heater &instance, float current, int8_t target);
+
+#if HAL_HAVE_IMU_HEATER2
+    struct imu_heater heater2;
```

Comment:
It might be better to do this as an array and have some kind of MAX_HEATER_INSTANCES variable - not sure. Maybe @peterbarker has an opinion